PublicMapping/districtbuilder-classic
# Fix CSRF error when uploading plan files

## Overview

Fixes a CSRF error when uploading plans. Also fixes a second error that was hidden by the first, related to changes to how Celery serializes objects by default.

### Checklist

- ~[ ] Files changed in the PR have been `yapf`-ed for style violations~ (I tried this but it just made two small changes that weren't actually problems, so I skipped it).

## Testing Instructions

 * Log in and go to "Upload Plan"
 * Choose a file and upload it. If you want to see a complete success on the next step, either grab one of the index files from the sample Virginia data zip file, or request an index file for an existing template plan. Otherwise, you can just upload any old file.
 * If you uploaded a plan index file, you should see a note about your file being processed. If you uploaded some other kind of file, you should see a message that the file wasn't able to be processed. You shouldn't see a CSRF error, nor should you see a Serialization Error.
